package.json

```diff
@@ -25,12 +25,11 @@
     "ember-cli-import-polyfill": "^0.2.0",
     "ember-cli-is-package-missing": "^1.0.0",
     "ember-cookies": "^0.0.11",
-    "ember-getowner-polyfill": "^1.1.0",
```

Comment:
ESA supports Ember version starting from 1.12 so we need the polyfill for those versions.